vulture_os/toolkit/api_parser/cisco_meraki/cisco_meraki.py

```diff
@@ -113,7 +113,7 @@ def test(self):
 
     def get_logs(self, network_id, product_type, since):
         self._connect()
-        logger.debug(f"[{__parser__}]:get_logs: Get user events request params: startingAfter={since}",
+        logger.info(f"[{__parser__}]:get_logs: Get user events request params: startingAfter={since}",
```

Comment:
```suggestion
        logger.info(f"[{__parser__}]:get_logs: Get user events for network ID {network_id}, product type {product_type}, starting after {since}",
```